homework_5/Methods/GuessWordGame.java

```diff
@@ -1,5 +1,35 @@
 package Methods;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class GuessWordGame {
-    
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        Random random = new Random();
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado" , "broccoli", "carrot", "cherry",
+                "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", " pea", "peanut",
+                "pear", "pepper", "pineapple", "pumpkin", "potato"};
+        String randomWord = words[random.nextInt(words.length)];
+        System.out.println(Arrays.toString(words));
+        System.out.print("Guess a word: ");
+        String guess_word = scanner.nextLine().toLowerCase();
```

Comment:
It would be better if you follow some naming rules, like the camel case. It's not allowed in Java to name a variable using "_", so let's rename 'guess_word' to 'guessWord'